packages/components/Button/src/Button.styling.ts

```diff
@@ -95,6 +96,22 @@ export const getDefaultSize = (): ButtonSize => {
   return 'medium';
 };
 
+export const getPlatformSpecificAppearance = (appearance: ButtonAppearance): ButtonAppearance => {
+  switch (appearance) {
+    case 'ascent': // Included to cover Mobile platform naming guidelines, maps to 'primary'.
+      return 'primary';
+    case 'outline': // Exists only for Mobile platforms, maps to default on other platforms.
+      if (Platform.OS == 'android' || Platform.OS == 'ios') return 'outline';
```

Comment:
> return 'outline'; [](http://example.com/codeflow?start=59&length=18)

styling nit: For consistency and readability, all conditional bodies should be wrapped in curly braces and bodies should be on their own line.